examples/vision/mmrazor_pruning/example.md

```diff
@@ -0,0 +1,75 @@
+7. [Results](#results)
+#### 1. Downloading NyunCLI
+Nyun-cli offers the users the luxury to compress their neural networks via a single command line. To download and install NyunCLI.
+```bash
+pip install nyuncli
+nyun init {WORKSPACE_PATH} ""
+```
+Here ``WORKSPACE_PATH`` is the root folder for running the experimentation and ``CUSTOM_DATA_PATH`` defines the path of custom data. The ``YAML_PATH`` is the path of configuration to be used. The base configuration used for this example can be found at [vision scripts]().This example offers a basic example for  installing nyuncli visit nyuncli documentation for a more advanced installation.
+#### 2. Downloading the dataset
+Downloading and formatting CIFAR-10 dataset is automated via Nyuntam. We need to specify the following hyperparameters in the YAML configuration 
+```yaml
+DATASET_NAME: COCODETECTION
+DATASET_PATH: {DATASET_PATH}
+TASK: object_detection
+``` 
+Note: Dataset Path is expected to to be the path relative to WORKSPACE_PATH
+#### 3. Model Loading
+Nyuntam supports object detection models from MMDet and MMYolo libraries, we load YOLOX-tiny from MMDet for this experiment. Pretrained COCO weights and model config are internally downloaded during job run. NOTE: the model name argument must match the exact name used in mmdet configs. 
+```yaml
+MODEL_NAME: yolox_tiny_8xb8-300e_coco
+PLATFORM: mmdet
+```
+#### 4. Loading Pretrained Weights (Optional)
+You can optionally load pretrained weights if you already have the same.  This can be used to prune mmdet and mmyolo models trained on custom datasets. 
+```yaml
+CUSTOM_MODEL_PATH: {CUSTOM_MODEL_PATH}
+#leave CUSTOM_MODEL_PATH as "" if unused
+```
+#### 5. Setting Task Specific Hyperparameters
+We specify the hyperparameters used for running this tutorial below, we may modify them to better suit your pruning task, more details on the hyperparameters can be found at [Nyun Documentation](https://nyunai.github.io/nyun-docs/kompress/algorithms/).
+```yaml
+    INTERVAL: 10 # The interval between two pruning operations
+    NORM_TYPE: 'act' # Type of importance , activation or flops
+    LR_RATIO: 0.1 # Learning Rate Ratio
+    TARGET_FLOP_RATIO: 0.9 # Target Number of Flops value * number of existing flops.
+    EPOCHS: 5 # Total Number of Epochs to perform pruning (stops early once reached required flops)
+```
+#### 6. Starting the Job
+The following command starts the job using nyun-cli. 
+```bash
+nyun run {CONFIG_PATH}
+```
+``CONFIG Path`` Config path is the edited yaml used for defining the hyperparameters of the job. 
+
+After the job is completed the folder structure at Workspace_path will be as follows
+```
+jobs/
+	└──  mds.pt
+	└──  flops_{target_flop_ratio}.pth
+	└──  fix_subnet.json
+logs/
+	└── log.log
+```
+	
+####  7. Results
+
+We use MAP and Flops to assess the result produced. The results are as follows:
+	|Model | MAP | Latency|
+   |Yolox-tiny|32.0|0.3515 ms|
+   |YoloX-tiny pruned| 29.6| 0.3170 ms
```

Comment:
1. (nits) we can use 'tree <path>' to get the tree structure of the output dir for a cleaner look
2. the results table isn't correctly formatted (lets use https://www.tablesgenerator.com if required) 
<img width="1168" alt="image" src="https://github.com/user-attachments/assets/c1250421-390f-481c-8144-ca96deef6506">